index.html

```diff
@@ -49,26 +75,25 @@ <h2>Our Mission</h2>
   
   <p>Recipients should be able to easily access the content of every email sent to them, regardless of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation and sending tools, and email clients to drive this vision forward at every step of the email journey.</p>
   
-  <h2>Quick Links</h2>
+  <h2>Quick Links</h2><img class="new-tab" src="img/new-tab-icon.png" alt="new tab icon">
   
   <ul>
+    
     <li>
-      <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/mission.md">
-        Mission statement
+      <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/mission.md" target="_blank">
```

Comment:
Do all external links need `target="_blank"`? I agree with adding it to the links to the forms/social media, but I'm unsure about adding this when linking to the EMC GitHub repo.

What do you think @dylanatsmith, @M-J-Robbins, @alicemeichi?

Reply:
Generally speaking, I'm against `target="_blank"` in the majority of cases. 
It's removing the choice from the user, so it's bad for user experience.
I agree, remove it from EMC GitHub links, but I would also go a step further a remove it from social links too.

Reply:
Opening external links to docs that don't live on the site in a new tab would allow users to return to the homepage and continue browsing without having to click on the back button.

But if everyone is against that, I can remove the option